shared/discord.py

```diff
@@ -29,12 +30,18 @@ def discordError(title, message=None):
         response = webhook.execute()
         return response.json().get('id')
 
-def discordUpdate(title, message=None, color=3066993, message_id=None):
+def discordUpdate(title, message=None, color=3066993, message_id=None, content=None, timestamp=None):
     embed = DiscordEmbed(title=title, description=message, color=color)
+    embed.set_author(name="Blackhole", icon_url="https://cdn.discordapp.com/attachments/1222893572521594962/1262384669161295934/black-hole-556660.png?ex=669666d7&is=66951557&hm=e792b1162ea86c1ba24ee9f4568c571ff1e3429e5b295555fea3fcb94404b434&")
```

Comment:
discordUpdate is also used for other scripts, so we should probably do this more dynamically